en_US/changes/changes-ee-5.2.md

```diff
@@ -1,5 +1,19 @@
 # Releases
 
+
+## e5.2.1
+
+### Enhancements
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) Added support for defining templates for MQTT 5.0 publish properties and user properties in Republish rule action.
+
+- [#11612](https://github.com/emqx/emqx/pull/11612) During node evacuation, evacuate all disconnected sessions, not only those started with `clean_start` set to `false`.
+
+### Bug Fixes
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) Fixed an issue where an ill-defined built-in rule action config could be interpreted as a custom user function.
```

Comment:
Should we add this one? https://github.com/emqx/emqx/pull/11630

Reply:
Done, @Meggielqk please help to add translated version